lighthouse-core/config/config-plugin.js

```diff
@@ -207,11 +229,15 @@ class ConfigPlugin {
       audits: pluginAuditsJson,
       category: pluginCategoryJson,
       groups: pluginGroupsJson,
+      locales: pluginLocalesJson,
       ...invalidRest
     } = pluginJson;
 
     assertNoExcessProperties(invalidRest, pluginName);
 
+    // Not part of Config.Json, only validated.
```

Comment:
Currently there is a 1:1 mapping between plugin config and core config. As the `locales` key is only supported for plugins I figured it would be better to not include it in the core config to avoid confusion. With this method we are validating here and then just reading from the raw JSON after assuming nothing is thrown.